src/lib/CustomRulePathManager.ts

```diff
@@ -58,15 +65,15 @@ export class CustomRulePathManager implements RulePathManager {
 		}
 
 		// Now that we've got the file contents, let's turn it into a JSON.
-		const json = JSON.parse(data);
+		const json: RulePathJson = JSON.parse(data) as RulePathJson;
```

Comment:
Seems verbose to define the type at both ends.

Reply:
Turns out only the latter definition is necessary.